### ISSUE TITLE
Add support to phpactor

### ISSUE THREAD

**Issue by afranioce** [kabouzeid/nvim-lspinstall#114]

**Comment by kabouzeid** [comment]
how does this compare to intelephense?

**Comment by afranioce** [comment]
Yes, the phpactor is 100% free and open-source, the performance is similar, the intelephense is not 100% free and isn't open-source, phpactor has support to Go to type definition, rename, and Code actions, the intelephense is paid :)
